### PR TITLE
Gemspec update to alleviate dependancy collision

### PR DESCRIPTION
**Pull request by travega**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kant01ne) [inline .idea/vcs.xml]: can you remove that file ?

Comment (travega) [reply .idea/vcs.xml]: How did that get in there?!! Yeh I'll get rod of it now.

**Review by kant01ne — APPROVED** [review]
LGTM

**Comment by kant01ne** [comment]
LGTM!
Same process as here to release : 
https://github.com/intercom/intercom-ruby-app/blob/master/RELEASING

**Pull request merged**